api_types.ts

```diff
@@ -599,7 +599,11 @@ export interface TemporaryBlobStorage {
    * If the `downloadFilename` parameter is specified, the data will be interpreted as a file (`attachment` content
    * disposition) that will be downloaded when accessed as the file name provided.
    */
-  storeUrl(url: string, opts?: {expiryMs?: number; downloadFilename?: string}): Promise<string>;
+  storeUrl(
+    url: string,
+    opts?: {expiryMs?: number; downloadFilename?: string},
+    fetchOpts?: Pick<FetchRequest, 'disableAuthentication'>,
```

Comment:
Could potentially allow anything from `FetchRequest`, but for now `disableAuthentication` was all I needed.

It might be ideal if the first argument here was just a fetch request but it wouldn't be backwards compatible

If we imagine a future where the `fetch` replaces the `url` then it might make sense to name it `fetch` instead of `fetchOpts`

Reply:
Great, yeah this is forward-compatible with adding more options, so seems fine to start here.